src/pages/[...slug].tsx

```diff
@@ -119,7 +120,15 @@ export const getStaticProps: GetStaticProps<Props, Params> = async (
   let layout = frontmatter.template
 
   if (!frontmatter.template) {
-    layout = params.slug.includes("developers/docs") ? "docs" : "static"
+    layout = params.slug.join('.').includes("developers/docs") ? "docs" : "static"
+  }
+
+  if (params.slug.join('/').includes("developers/tutorials")) {
```

Comment:
Can we put this condition inside the previous one and do something like this to improve readability?

```ts
if (!frontmatter.template) {
  layout = "static"

  if(params.slug.includes("developers/docs")) {
    layout = 'docs'
  }

  if(params.slug.includes("developers/tutorials")) {
    layout = "tutorial"
  }
}
```